components/api/flow.js

```diff
@@ -35,6 +39,10 @@ const Flow = ({ url }) => (
     <SectionLayout sub title="Installing the definitions">
       <CodeBlock code={install} />
     </SectionLayout>
+  
+    <p>If you are on npm >= 5.2 you could simply use npx</p>
```

Comment:
Ok fair enough :) Maybe we can use this article then? https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b